app/models/package_manager/base.rb

```diff
@@ -125,12 +125,6 @@ def self.update(name, sync_version: :all)
 
       save_dependencies(mapped_project, sync_version: sync_version) if self::HAS_DEPENDENCIES
       finalize_db_project(db_project)
-    rescue StandardError => e
-      if ENV["RACK_ENV"] == "production"
-        Bugsnag.notify(e)
-      else
-        raise
-      end
```

Comment:
I think this might've been added before it was run async? But should be fine to remove it AFAICT.